react/src/pages/utils/components/InfoList.tsx

```diff
@@ -65,6 +71,16 @@ export default function InfoList(props: { infoList: Info[]; title?: string; icon
                                     collapsibleTitles={info.collapsibleTitles}
                                     collapsibleValues={info.collapsibleValues}
                                 />
+                                {props.linkPath && info.identifier &&
+                                    <Button
+                                    href={`${props.linkPath}/${info.identifier}`}
```

Comment:
You should use the react-router's history API to push a state here, as a regular hyperlink like this causes a full page reload.